Makefile

```diff
@@ -36,9 +36,12 @@ serve-d: .state/docker-build-$(APP_ENV)
 serve: .state/docker-build-$(APP_ENV)
 	$(DOCKER_COMPOSE) up
 
-debug: .state/docker-build-$(APP_ENV)
+shell: .state/docker-build-$(APP_ENV)
 	$(DOCKER_COMPOSE) run --service-ports web python -m measurements shell
 
+create-tables: .state/docker-build-$(APP_ENV)
+	$(DOCKER_COMPOSE) run web python -m measurements create_tables
+
```

Comment:
Ah, I understand: now to create tables one needs to use the specific command. This has probably been done to avoid the warning that every worker was trying to create tables.